### PR TITLE
Add extra description for beginner contributors

### PR DESCRIPTION
**Pull request by seun-beta**
Added extra description for beginner contributors. Also fixed some typos. I was thinking of making the tone of the docs more friendly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,22 +7,33 @@ This document outlines a generic process of contributing and applies to all CHAO
...
 - Mailing Lists
 - Community calls
 - Working group calls
 - GitHub
 
+## Catch us online
+
+- [CHAOSS on the Web](https://chaoss.community)
+- [CHAOSS on GitHub](https://github.com/chaoss)
+- [CHAOSS on Community Bridge](https://tinyurl.com/y6bao886)
```
Comment (GeorgLink) [inline CONTRIBUTING.md:24]:
```suggestion
- [CHAOSS on LFX Crowdfunding](https://crowdfunding.lfx.linuxfoundation.org/projects/a4a43b66-f707-47b4-97cc-484285c274a8)
- [CHAOSS on Open Collective](https://opencollective.com/chaoss) 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -68,7 +79,14 @@ Then, [submit a pull request][pr] on GitHub to the CHAOSS repository.
 [pr]: https://github.com/thoughtbot/factory_girl_rails/compare/
 [clone]: https://help.github.com/articles/cloning-a-repository/
 
-At this point you are waiting on the CHAOSS repository maintainers. They will comment on your pull requests
+### Pull Request Checklist  
+
+- [x] Check if your code addition will pass both code linting checks and unit test.  
+- [x] Check if your documentation commits are free from typos.
+- [x] Check the commits and commits messages and ensure they are free from typos.
+- [x] Please describe your pull request.
```
Comment (GeorgLink) [inline CONTRIBUTING.md:87]:
```suggestion
- [ ] Check if your code addition will pass both code linting checks and unit test.  
- [ ] Check if your documentation commits are free from typos.
- [ ] Check the commits and commits messages and ensure they are free from typos.
- [ ] Please describe your pull request.  
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @seun-beta. This is great. I have a few change requests.

**Comment by seun-beta** [comment]
Hello @GeorgLink I have committed the suggested changes. Apologies for the late response, I was on Easter break.

**Review by GeorgLink — APPROVED** [review]
Thanks @seun-beta 

I hope you had a great Easter break

**Pull request merged**